operators/tokenizer/sentencepiece_tokenizer.cc

```diff
@@ -32,35 +33,49 @@ void KernelSentencepieceTokenizer::Compute(const ortc::Tensor<std::string>& inpu
                                            bool add_rev,
                                            ortc::Tensor<int32_t>& output,
                                            ortc::Tensor<int64_t>& output1,
+                                           std::optional<ortc::Tensor<int32_t>*> output2,
```

Comment:
Operator inputs/outputs are based on their order so I believe this needs to be added at the end of current arguments otherwise it will break any existing usage.

e.g. any existing model where the `fairseq` output is being consumed will expect the 3rd output to be a scalar bool and this change would break that model

Reply:
Thanks. Done